src/app/shared/comments/comments.component.scss

```diff
@@ -123,3 +123,55 @@
   margin: 0 1rem;
   color: #aaaaaa;
 }
+
+body {
+  > app-root {
+    > app-shell {
+      > div {
+        > mat-sidenav-container {
+          > mat-sidenav-content {
+            > app-view-project {
+              > div {
+                > div.panel-main {
+                  > app-issue {
+                    > div {
+                      > div.panel-main__issues.ng-star-inserted {
+                        > app-detached-issue {
+                          > div {
+                            > div.issue-comments {
+                              > div {
+                                > app-comments {
+                                  > div {
+                                    > div.new_comment.ng-star-inserted {
```

Comment:
You don't need to specify upto this level of depth.

Reply:
Also, use this CSS in the `styles.scss` file such that it can be used throughout the project. 
Remember to remove previous definitions (if any)